Source/Shaders/PostProcessStages/AcesTonemapping.glsl

```diff
@@ -17,16 +17,6 @@ void main()
 #ifdef AUTO_EXPOSURE
     color /= texture2D(autoExposure, vec2(0.5)).r;
 #endif
-    float g = 0.985;
```

Comment:
Should the above comment/reference get moved to the new file?  This file isn't doing much now, right?